AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.scss

```diff
@@ -7,6 +7,7 @@
   padding: 1em 0;
 
   .judge-row,
+  .staff-member-row,
   .participant-row {
```

Comment:
Is there any other types of rows or could this be a common class?

Reply:
I had this in case any custom css needed to be applied to the staff member row, but there wasn't so I'm changing it to use the partipant row.

Reply:
I added this in case the SM needed it's own css but it didn't in the end, so I've changed it to use the participant row.